pom.xml

```diff
@@ -27,12 +27,45 @@
     <version>2.0.24</version>
     <name>WSO2 Carbon - Mediation Library Connector For fileconnector</name>
     <url>http://wso2.org</url>
+    <profiles>
+        <profile>
+            <id>EI6XX</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+                <property>
+                    <name>env</name>
+                    <value>EI6XX</value>
+                </property>
+            </activation>
+            <properties>
+                <product.name>wso2ei</product.name>
+                <product.version>6.5.0</product.version>
+                <management.port>9763</management.port>
+                <startup.script>integrator</startup.script>
+            </properties>
+        </profile>
+        <profile>
+            <id>EI7XX</id>
+            <activation>
+                <property>
+                    <name>env</name>
+                    <value>EI7XX</value>
+                </property>
+            </activation>
+            <properties>
+                <product.name>wso2mi</product.name>
+                <product.version>1.2.0-beta</product.version>
```

Comment:
For what we need this version number? 
This will change from one pack to another.